src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -0,0 +1,156 @@
+import static nextstep.subway.utils.JsonPathUtils.*;
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.subway.utils.DatabaseCleaner;
+import nextstep.subway.utils.JsonPathUtils;
+
+@DisplayName("지하철 노선 관련 기능")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class LineAcceptanceTest {
+    private static final String LINE_2 = "2호선";
+    private static final String LINE_BUNDANG = "분당선";
+    @LocalServerPort
+    int port;
+    @Autowired
+    DatabaseCleaner databaseCleaner;
+    private Integer GANGNAM_ID;
+    private Integer WANGSIPLI_ID;
+    private Integer JUKJUN_ID;
+
+    @BeforeEach
+    public void setUp() {
+        if (RestAssured.port == RestAssured.UNDEFINED_PORT) {
+            RestAssured.port = port;
+        }
+        databaseCleaner.execute();
+
+        GANGNAM_ID = extractInteger(지하철역_생성(GANGNAM), "$.id");
+        WANGSIPLI_ID = extractInteger(지하철역_생성(WANGSIPLI), "$.id");
+        JUKJUN_ID = extractInteger(지하철역_생성(JUKJUN), "$.id");
+    }
+
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철 노선 생성")
+    @Test
+    void createStationLine() {
+        // when
+        지하철_노선_생성(LINE_2, GANGNAM_ID, WANGSIPLI_ID);
+
+        // then
+        ExtractableResponse<Response> response = 지하철_노선_목록_조회();
+
+        assertAll(
+            () -> assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value()),
+            () -> assertThat(extractList(response, "$[*].name")).contains(LINE_2),
+            () -> assertThat(extractList(response, "$[*].stations[*].name")).contains(GANGNAM, WANGSIPLI)
+        );
+    }
+
+    /**
+     * Given 2개의 지하철 노선을 생성하고
+     * When 지하철 노선 목록을 조회하면
+     * Then 지하철 노선 목록 조회 시 2개의 노선을 조회할 수 있다.
+     */
+    @DisplayName("지하철노선 목록 조회")
+    @Test
+    void getStationLines() {
+        // given
+        지하철_노선_생성(LINE_2, GANGNAM_ID, WANGSIPLI_ID);
+        지하철_노선_생성(LINE_BUNDANG, WANGSIPLI_ID, JUKJUN_ID);
+
+        // when
+        ExtractableResponse<Response> response = 지하철_노선_목록_조회();
+
+        // then
+        assertAll(
+            () -> assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value()),
+            () -> assertThat(extractList(response, "$[*].name")).contains(LINE_2, LINE_BUNDANG),
+            () -> assertThat(extractList(response, "$[*].stations[*].name")).contains(GANGNAM, WANGSIPLI, JUKJUN)
```

Comment:
```suggestion
            () -> assertThat(역목록_추출(response).contains(GANGNAM, WANGSIPLI, JUKJUN)
```

가독성을 위해 좀 더 메소드로 분리시켜도 좋을것 같아요!

Reply:
다음 스탭에 반영하도록 하겠습니다!